lib/themes/dosomething/paraneue_dosomething/js/url/helpers.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * Get a specified segment from the pathname.
+ *
+ * @param  {String} pathname
+ * @param  {Number} segment
+ * @return {Bool}
+ */
+export function getPathnameSegment(pathname = window.location.pathname, segmentIndex = 0) {
+  let segments = pathname.split('/');
+
+  segments.splice(0,1);
```

Comment:
Yeah. Was worried that would muddle things, but I guess not really since not returning the full array of segments and just a single piece.